jupyterhub/templates/image-puller/serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+{{- /*
+ServiceAccount for the pre-puller hook
```

Comment:
```suggestion
ServiceAccount for the pre-puller hook's image-awaiter-job
```

Reply:
Rebased it into a previous commit (the one where `serviceaccount.yaml` was created)